resources/inspec.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 
 provides :chef_inspec
```

Comment:
can we just inspec for provides and as a resource name instead of `chef_inspec`?